### DISCUSSION TITLE
Nombres différents sur la page Stats

### DISCUSSION THREAD

**Discussion by ghost** [betagouv/ma-cantine#1782] [Idées]
Suite à la remarque d'une utilisatrice ce jour et à ma propre incompréhension initiale devant la différence de chiffre entre le nombre indiqué dans l'encart de gauche et la somme des nombres donnés dans la répartition par secteur à droite sur la page Statistiques (cf. capture d'écran en PJ), je propose l'une de ces deux choses :

1. une solution facile : ajouter, sous la répartition par secteur, une formule du type : "⚠️ Si le nombre donné à gauche est différent de la somme de ceux que vous voyez ci-dessus, c'est qu'une même cantine peut opérer dans plusieurs secteurs à la fois."

2. une solution plus chronophage, mais plus propre : attribuer aux cantines un "secteur principal" et, si elles en ont, des "secteurs secondaires". De ce que je constate, les cantines ont en effet un type de convives majoritaire. Par exemple, une cantine de lycée accueille principalement des lycéens, mais aussi minoritairement des collégiens. Nous n'aurions plus ainsi qu'à afficher la répartition par secteur principal, ce qui me paraît plus clair pour l'utilisateur. Évidemment cette solution pose la question de savoir ce que l'on fait pour déterminer le secteur principal des cantines qui ont plusieurs secteurs d'activités parmi les 9000+ déjà existantes.

Votre avis, @JenniferStephan, @hfroot et @alemangui?
@gauthierandre, @Greta712, Pauline, avez-vous eu des commentaires similaires?

<img width="1118" alt="Capture d’écran 2022-08-16 à 12 37 08" src="https://user-images.githubusercontent.com/93709421/184892641-9d010237-6e83-499c-8f6f-242b871ae273.png">

**Comment by alemangui** [comment]
Je serai pour la première solution, plus simple et claire.

**Reply by ghost** [reply]
D'ac.

**Comment by valeriemerle** [comment]
Je partage cette remarque de lisibilité et j'ai dejà eu ce type d retour. Ne pas ecarter la seconde car ce que dit JB est vrai souvent un secteur principal. La solution devrait venir aussi dans le temps ce probleme de multi secteur est lié aux cuisines centrales alors que les cuisines sites ou satellite n'ont qu'un secteur

**Reply by ghost** [reply]
Figure-toi que même les cuisines-cantines peuvent avoir, même si c'est rare, plusieurs secteurs. Ex. : une cantine de lycée qui accueille aussi des collégiens comme convives => "Secondaire lycée (hors agricole)" + "Secondaire collège"
